.github/workflows/build-test.yml

```diff
@@ -0,0 +1,32 @@
+name: Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    name: Build & test
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: yarn --frozen-lockfile --ignore-scripts
+    - run: yarn build
+    - run: yarn lint
+    - run: yarn test
+  all-jobs-pass:
+    name: All jobs pass
+    runs-on: ubuntu-20.04
+    needs:
+      - build
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
Could this step me omitted? :thinking: I don't see much utility in checking out the repo here.

Reply:
There needs to be _some_ step and this seems like a harmless empty step

Reply:
OK, fair enough. I wasn't aware a step was required. We can replace this later with something more lightweight if need be.